zbuild.h

```diff
@@ -1,6 +1,34 @@
 #ifndef _ZBUILD_H
 #define _ZBUILD_H
 
+/* Determine compiler version of C Standard */
+#ifdef __STDC_VERSION__
+#  if __STDC_VERSION__ >= 199901L
+#    ifndef STDC99
+#      define STDC99
```

Comment:
`STDC99` is also *defined* in zconf-ng.h.in and zconf.h, so do we also need to define `STDC11` there too?